packages/nuxt/src/module.ts

```diff
@@ -71,6 +72,21 @@ export default defineNuxtModule<ModuleOptions>({
       })
     })
 
+    // add scalar tab to DevTools
+    if (_nuxt.options.dev && _options.devtools) {
+      _nuxt.hook('devtools:customTabs', (tabs) => {
+        tabs.push({
+          name: 'scalar',
+          title: 'Scalar',
```

Comment:
one other thing that I forgot to mention, do you want to add `scalar` to `server` category? cc @amritk

Reply:
yep that sounds like a good spot for it!